shared/logutil/logutil.go

```diff
@@ -28,3 +30,28 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+//Masks the url credentials before logging for security purpose
+//[scheme:][//[userinfo@]host][/]path[?query][#fragment] -->  [scheme:][//[***]host][/***][#***]
+//if the format is not matched nothing is done, string is returned as is.
+func MaskCredentialsLogging( currUrl string) string {
+	//error if the input is not a URL
+	MaskedUrl := currUrl
+	u, err := url.Parse(currUrl)
+	if err != nil {
+		//log.Fatal(err)
```

Comment:
```suggestion
```

Reply:
Should i create a new file logutil_test.go  ?  and add the test cases there?